addons/profile/profile.php

```diff
@@ -201,21 +207,36 @@ public function user_pages() {
 	/**
 	 * Output user profile menu.
 	 *
-	 * @param int|false $user_id Id of user, default is current user.
-	 * @param string    $class_name   CSS class.
+	 * @param int|false $user_id    Id of user, default is current user.
+	 * @param string    $class_name CSS class.
 	 */
 	public function user_menu( $user_id = false, $class_name = '' ) {
 		$user_id     = false !== $user_id ? $user_id : ap_current_user_id();
-		$current_tab = get_query_var( 'user_page', ap_opt( 'user_page_slug_questions' ) );
+		$user        = get_user_by( 'id', $user_id );
```

Comment:
@mebishalnapit Why did you removed `ap_opt( 'user_page_slug_questions' )` ?

Reply:
@rahularyan Currently, I do not remember it. However it works with addition of that piece of code so I re-added it.